app/api/event_invoices.py

```diff
@@ -52,6 +53,13 @@ class Meta:
                          related_view_kwargs={'event_invoice_id': '<id>'},
                          schema='EventSchema',
                          type_='event')
+    discount_codes = Relationship(attribute='discount_code',
```

Comment:
Shouldn't this be `discount_code` ? since event_invoice model has one `discount_code_id` ... So one-to-one ?